server/src/assets/game/script-game.js

```diff
@@ -11,8 +11,41 @@ var isEasyMode = false;      // User interface verbosity.
 const numberOfDestinationsMax = 12;           // Maximum destinations to display
 const destinationNamePrefix = "destination";  // HTML element ID prefix of the destination buttons
 
-var allPossibleDestinations = null;           // Platform specific lookup table for destinations
-var signalFlagMap = null;                     // Platform specific lookup table for signal flags
+// Initialisation of Lookup tables
+var allPossibleDestinations = null;  // Global Access variable for destination lookup table
+var signalFlagMap = null; // Global access variable for signal flag mapping
+
+// Tries to determine the name of the SWTbahn Platform which is being used by querying the server.
+// If the server does not reply with the platform name, retry after some delay.
+// If the server does reply with the platform name, tries to load the destinations and the flags 
+// mapped for the respective platform. If this fails, does not perform a retry.
+function tryLoadDestinationsAndFlagMap() {
```

Comment:
Test if this works if a) server is not started up initially and b) server is already started when opening client webpage.